src/auth/auth-token.dto.ts

```diff
@@ -15,8 +15,8 @@ export class AuthTokenDto {
   createdAt: Date;
   @IsDate()
   @IsOptional()
-  validUntil: Date;
+  validUntil: Date | null;
```

Comment:
Looking at the logic of the AuthToken creation. This shouldn't be optional, because either the user requested a validUntil Date or it's good for two years…

Reply:
I'd like to keep changes in this PR focused on adjusting the types to the current database schema. Can we push this change to another PR?